docs/api/command-line-switches.md

```diff
@@ -67,13 +67,25 @@ Enables caller stack logging for the following APIs (filtering events):
 
 * `desktopCapturer.getSources()` / `desktop-capturer-get-sources`
 
-### --enable-logging
+### --enable-logging[=file]
 
-Prints Chromium's logging into console.
+Prints Chromium's logging to stderr (or a log file).
 
-This switch can not be used in `app.commandLine.appendSwitch` since it is parsed
-earlier than user's app is loaded, but you can set the `ELECTRON_ENABLE_LOGGING`
-environment variable to achieve the same effect.
+This switch can not be used in `app.commandLine.appendSwitch` since it is read
```

Comment:
Should be adjusted a little that we support for the first tick ?

Reply:
ah yep, I think I fixed this somewhere else but missed this one. 👍